app/discord/draw.js

```diff
@@ -48,9 +48,16 @@ export default async function draw(message, opt) {
   }
 
   log.debug(`PGBG: ${message.author.username} drew the ${card.name} from ${opt} - ${card.image}`);
-    
   // build the message
-  const image = await images.draw(`./assets/decks/${opt.toLowerCase()}/${card.image}`, reversed ? 180 : 0);
+  var image;
```

Comment:
`var` is seldom used in modern JS. `var` is only used when you need to declare a variable that's accessible in the parent function scope. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#scoping_rules_2 for a complete explanation.

`let` would be most appropriate here.